herald/src/test/java/com/vmware/herald/sensor/datatype/LocationReferenceTests.java

```diff
@@ -46,4 +46,18 @@ public void testWGS84PointLocationReference() {
         assertNotNull(new WGS84PointLocationReference(null, null, null).description());
         assertNotNull(new WGS84PointLocationReference(1d, 2d, 3d).description());
     }
+
+    @Test
+    public void testInertiaLocationReference() {
```

Comment:
Appears to be sunny day testing only. Please test extremes of values and values that together do not make sense. Also please test negative values.